pkgs/top-level/all-packages.nix

```diff
@@ -4431,6 +4431,9 @@ in
 
   munge = callPackage ../tools/security/munge { };
 
+  munt = libsForQt5.callPackage ../applications/audio/munt { };
+  munt_lv2 = callPackage ../applications/audio/munt/lv2.nix { };
```

Comment:
Wouldn't it be better to use `munt-lv2`?

Reply:
@matthiasbeyer Fixed, thanks!